app/build.gradle

```diff
@@ -59,6 +59,12 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.3.1'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
 
+    // koin
+    def koin_version = "3.1.1"
+    implementation "io.insert-koin:koin-android:$koin_version"
```

Comment:
implementation "io.insert-koin:koin-android:$koin_version" I think this is enough. Not sure why you need the workManager here?